test/cli/test-other.py

```diff
@@ -859,3 +860,27 @@ def test_build_dir_j_memleak(tmpdir): #12111
     ]
 
     assert_cppcheck(args, ec_exp=0, err_exp=[], out_exp=out_lines)
+
+
+def test_premium_with_relative_path(tmpdir):
+    product_name = 'Cppcheck Premium ' + str(time.time())
+
+    test_cfg = tmpdir.join('cppcheck.cfg')
+    with open(test_cfg, 'wt') as f:
+        f.write("""
+                {
+                    "addons": [],
+                    "productName": "NAME",
+                      "about": "Cppcheck Premium 1.2.3.4"
+                }
+                """.replace('NAME', product_name))
+
+    args = ['--premium=misra-c++-2008', 'test.c']
```

Comment:
Please clarify why you execute with these args in a comment

Reply:
if `isCppcheckPremium()` returns false then `--premium` options will generate error output